tsconfig.json

```diff
@@ -24,6 +24,7 @@
     "allowSyntheticDefaultImports": true,
     "noFallthroughCasesInSwitch": true,
     "resolveJsonModule": true,
+    "noEmit": true
```

Comment:
I see this thing `noEmit: true` appearing everywhere, that prevents MD from being started on Docker and Netlify. It's created by `create-react-app` while running `start` script. 
I wonder, were you able to run MD in Docker locally?

Reply:
Let's not commit, there should be a workaround: https://github.com/Exabyte-io/materials-designer/commit/7069296c51125075769527d79cb68d53945ae36e